travis/test_server.py

```diff
@@ -260,16 +261,19 @@ def setup_server(db, odoo_unittest, tested_addons, server_path, script_name,
         pipe = subprocess.Popen(cmd_odoo,
                                 stderr=subprocess.STDOUT,
                                 stdout=subprocess.PIPE)
-        for line in iter(pipe.stdout.readline, ''):
-            if line == b'':
-                break
-            if hidden_line(line, main_modules=[], addons_path_list=[],
-                           hidden_all_no_translation=True):
-                continue
-            # No show a warning from template runtime
-            # if 'openerp.models.schema' in line:
-            #     line = line.replace('DEBUG', 'WARNING')
-            print(print_log(line))
+        if stdout_log:
```

Comment:
Could you test with another project without configure if the log is printed?

Reply:
I tested with a gitlab project, modified MQT branch and showed me logs openerp_template, also I put a pdb and checked that was in this line printing logs.

Reply:
Could you share the URL of the pipeline, please?

NOTE: It should be different to Absa

Reply:
Of course,

https://git.vauxoo.com/vauxoo-dev/lodi/-/pipelines/111097

Regards.